[2024-07-30T10:26:59.543+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-07-30T10:26:59.633+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: pipelinev5.Run_data_quality_checks scheduled__2018-11-08T00:00:00+00:00 [queued]>
[2024-07-30T10:26:59.663+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: pipelinev5.Run_data_quality_checks scheduled__2018-11-08T00:00:00+00:00 [queued]>
[2024-07-30T10:26:59.665+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-07-30T10:26:59.773+0000] {taskinstance.py:2330} INFO - Executing <Task(DataQualityOperator): Run_data_quality_checks> on 2018-11-08 00:00:00+00:00
[2024-07-30T10:26:59.791+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=1036) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-07-30T10:26:59.794+0000] {standard_task_runner.py:63} INFO - Started process 1046 to run task
[2024-07-30T10:26:59.791+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'pipelinev5', 'Run_data_quality_checks', 'scheduled__2018-11-08T00:00:00+00:00', '--job-id', '138', '--raw', '--subdir', 'DAGS_FOLDER/pipeline_dag.py', '--cfg-path', '/tmp/tmpsbz2pzg3']
[2024-07-30T10:26:59.799+0000] {standard_task_runner.py:91} INFO - Job 138: Subtask Run_data_quality_checks
[2024-07-30T10:27:00.029+0000] {task_command.py:426} INFO - Running <TaskInstance: pipelinev5.Run_data_quality_checks scheduled__2018-11-08T00:00:00+00:00 [running]> on host c3b60429ae20
[2024-07-30T10:27:00.600+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='pipelinev5' AIRFLOW_CTX_TASK_ID='Run_data_quality_checks' AIRFLOW_CTX_EXECUTION_DATE='2018-11-08T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2018-11-08T00:00:00+00:00'
[2024-07-30T10:27:00.607+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-07-30T10:27:00.709+0000] {data_quality_operator.py:23} INFO - Checking Data Quality for songplays table
[2024-07-30T10:27:00.763+0000] {base.py:84} INFO - Using connection ID 'postgres_conn' for task execution.
[2024-07-30T10:27:01.488+0000] {sql.py:487} INFO - Running statement: SELECT COUNT(*) FROM songplays, parameters: None
[2024-07-30T10:27:01.493+0000] {sql.py:496} INFO - Rows affected: 1
[2024-07-30T10:27:01.495+0000] {data_quality_operator.py:30} INFO - Data quality on table songplays check passed with 1594 records
[2024-07-30T10:27:01.496+0000] {data_quality_operator.py:23} INFO - Checking Data Quality for users table
[2024-07-30T10:27:01.656+0000] {base.py:84} INFO - Using connection ID 'postgres_conn' for task execution.
[2024-07-30T10:27:01.966+0000] {sql.py:487} INFO - Running statement: SELECT COUNT(*) FROM users, parameters: None
[2024-07-30T10:27:01.972+0000] {sql.py:496} INFO - Rows affected: 1
[2024-07-30T10:27:01.976+0000] {data_quality_operator.py:30} INFO - Data quality on table users check passed with 75 records
[2024-07-30T10:27:01.980+0000] {data_quality_operator.py:23} INFO - Checking Data Quality for songs table
[2024-07-30T10:27:02.042+0000] {base.py:84} INFO - Using connection ID 'postgres_conn' for task execution.
[2024-07-30T10:27:02.178+0000] {sql.py:487} INFO - Running statement: SELECT COUNT(*) FROM songs, parameters: None
[2024-07-30T10:27:02.184+0000] {sql.py:496} INFO - Rows affected: 1
[2024-07-30T10:27:02.187+0000] {data_quality_operator.py:30} INFO - Data quality on table songs check passed with 71 records
[2024-07-30T10:27:02.188+0000] {data_quality_operator.py:23} INFO - Checking Data Quality for artists table
[2024-07-30T10:27:02.244+0000] {base.py:84} INFO - Using connection ID 'postgres_conn' for task execution.
[2024-07-30T10:27:02.262+0000] {sql.py:487} INFO - Running statement: SELECT COUNT(*) FROM artists, parameters: None
[2024-07-30T10:27:02.286+0000] {sql.py:496} INFO - Rows affected: 1
[2024-07-30T10:27:02.289+0000] {data_quality_operator.py:30} INFO - Data quality on table artists check passed with 69 records
[2024-07-30T10:27:02.290+0000] {data_quality_operator.py:23} INFO - Checking Data Quality for times table
[2024-07-30T10:27:02.320+0000] {base.py:84} INFO - Using connection ID 'postgres_conn' for task execution.
[2024-07-30T10:27:02.472+0000] {sql.py:487} INFO - Running statement: SELECT COUNT(*) FROM times, parameters: None
[2024-07-30T10:27:02.500+0000] {sql.py:496} INFO - Rows affected: 1
[2024-07-30T10:27:02.506+0000] {data_quality_operator.py:30} INFO - Data quality on table times check passed with 1507 records
[2024-07-30T10:27:02.511+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-07-30T10:27:02.575+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=pipelinev5, task_id=Run_data_quality_checks, run_id=scheduled__2018-11-08T00:00:00+00:00, execution_date=20181108T000000, start_date=20240730T102659, end_date=20240730T102702
[2024-07-30T10:27:03.045+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-07-30T10:27:03.258+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-07-30T10:27:03.331+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
